CHANGELOG.md

```diff
@@ -4,6 +4,24 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.11.2 (2023-08-09)
+
+This release contains bug fixes since the 2.11.0 release.
```

Comment:
```suggestion
This release contains bug fixes since the 2.11.1 release.
```

Reply:
I have removed #5909 , let's include it in the next minor release.